inventory/src/ts/components/item-group.ts

```diff
@@ -0,0 +1,115 @@
+/**
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import {Inventory, Item, itemType} from 'camelot-unchained';
+
+/**
+ * ItemGroup
+ */
+export class ItemGroup {
+  item: Item;
+  itemIDs: string[];
+  quantity: number;
+
+  constructor(item: Item) {
+    this.item = item;
+    this.quantity = 0;
+    this.itemIDs = [];
+    this.addItemID(item.id);
+  }
+
+  getFirstItemID(): string {
+    return this.itemIDs.length > 0 ? this.itemIDs[0] : null;
+  }
+
+  /**
+   * Check if the item group contains an item
+   * @param  {string} id - the id of item to look for
+   * @return {boolean} returns true if the item existing in the item group
+   */
+  hasItemID(id: string): boolean {
+    return this.itemIDs.filter((itemID: string) => {
```

Comment:
Given item IDs is a list of strings, rather than objects, would it not be simpler and more efficient to use:

    return this.itemIDs.indexOf(id) > -1;